EFCore/MAUI/Readme.md

```diff
@@ -0,0 +1,873 @@
+      try {
+         _photoBytes = await DataStore.GetAuthorPhotoAsync(Guid.Parse(itemId));
+         OnPropertyChanged(nameof(Thumbnail));
+         Post = await DataStore.GetItemAsync(itemId);
+         Id = Post.ID;
+         Title = Post.Title;
+         Content = Post.Content;
+         
+      }
+      catch (Exception e) {
+         System.Diagnostics.Debug.WriteLine($"Failed to Load Post {e}");
+      }
+   }
+   ```
+
+   _ViewModels/ItemsViewModel.cs_:
+
+   ```cs
+    public class ItemsViewModel : BaseViewModel {
+        Post _selectedPost;
+
+        public ItemsViewModel() {
+            // ...
+            ItemTapped = new Command<Post>(OnItemSelected);
+        }
+
+        public Command<Post> ItemTapped { get; }
+
+        public Post SelectedPost {
+            get => _selectedPost;
+            set {
+                SetProperty(ref _selectedPost, value);
+                OnItemSelected(value);
+            }
+        }
+
+        async void OnItemSelected(Post post) {
+            if (post != null) await Navigation.NavigateToAsync<ItemDetailViewModel>(post.ID);
+        }
+        // ...
+    }
+   ```
+
+   _App.xaml.cs_:
+
+   ```cs
+   public App() {
+      // ...
+      Routing.RegisterRoute(typeof(ItemDetailPage).FullName, typeof(ItemDetailPage));
+      // ...
+   }
+   ```
+   
+   _Views/ItemDetailPage.xaml_:
+
+   ```xml
+   <StackLayout Spacing="5" Padding="15">
+      <Label Text="Title:" FontFamily="Roboto" FontSize="12" TextColor="{StaticResource NormalLightText}"/>
+      <Label Text="{Binding Title}" FontFamily="Roboto" FontSize="14" TextColor="{StaticResource NormalText}" Margin="0, 0, 0, 15"/>
+      <Label Text="Content:" FontFamily="Roboto" FontSize="12" TextColor="{StaticResource NormalLightText}" />
+      <Label Text="{Binding Content}" FontFamily="Roboto" FontSize="14" TextColor="{StaticResource NormalText}"/>
+      <Label Text="Author:"></Label>
+      <Image   Source="{Binding Thumbnail}"></Image>
+   </StackLayout>
+   ```
+
+![](../../images/MAUI/DetailView.png)
+
+### Add a "New Item" entry form
+
+1. The Web API Service automatically generates OData endpoints required to create business objects. However, because the example application's security system is configured to disallow post creation for some users, it is beneficial to have a way to check a user's permissions before they can try to submit a post. To do this, implement a custom `CanCreate` endpoint.
+
+   _API/CustomEndPointController.cs_:
+   ```cs
+   [HttpGet(nameof(CanCreate))]
+   public IActionResult CanCreate(string typeName) {
+       var strategy = (SecurityStrategy)_securityProvider.GetSecurity();
+       var objectType = strategy.TypesInfo.PersistentTypes.First(info => info.Name  == typeName).Type;
+       return Ok(strategy.CanCreate(objectType));
+   }
+   ```
+
+2. Add a method that checks the permissions and a method that submits a new post to the .NET MAUI application's data service.
```

Comment:
Are we adding methods to the .NET Maui App service? Or are we submitting a post to the .NET Maui App service? 

I assume you mean add "methods to the service". Then again, I want to re-arrange thoughts so that related things go together. 

```suggestion
2. Add new methods to the .NET MAUI application's data service: one will check user permissions, another will  submit a new post.
```